### PR TITLE
User nav

### PR DESCRIPTION
**Pull request by WHomer**
What does this PR do?
satisfies #80.
<b>Notice</b> route for `get '/logout', to: 'sessions#logout'` was  needed for testing.

### CODE REVIEW

**Review by VinceCarollo — APPROVED** [review]

**Review by zachlp2016 — APPROVED** [review]
Excellent job looks gouda.

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -16,6 +14,10 @@ def create
     end
   end
 
+  def logout
+    reset_session
```
Comment (BrennanAyers) [inline app/controllers/sessions_controller.rb:18]: Is this the best/idiomatic way of doing this?
```For removing objects from the session, you can either assign a single key to nil:

# removes :person from session
session[:person] = nil
or you can remove the entire session with reset_session.
```
Answered my own question! From the Rails Docs

File: spec/features/navigation/application_spec.rb [hunk 2]
```diff
@@ -0,0 +1,150 @@
+require 'rails_helper'
+
+RSpec.describe 'within main navigation' do
+  context 'as a visitor' do
+    describe 'navbar includes the following' do
+      it 'a link to return to the welcome / home page of the application ("/")' do
+        visit root_path
```
Comment (BrennanAyers) [inline spec/features/navigation/application_spec.rb:7]: Do we want the Navbar on the Welcome page?

File: spec/features/users/new_spec.rb [hunk 3]
```diff
@@ -46,30 +46,10 @@
...
-        fill_in 'Zip', with: '80000'
-        fill_in 'Email', with: 'user_1@gmail.com'
-        fill_in 'Password', with: 'password'
-        fill_in 'Confirm password', with: 'password'
-
-        click_button 'Create User'
-
-
-
+        user_2 = User.create!(name: "User_1", role: 0, active: true, password_digest: "8320280282", address: "333", city: "Denver", zip: "80000", email: "user_1@gmail.com", state: 'IL' )
```
Comment (BrennanAyers) [inline spec/features/users/new_spec.rb:49]: Nice, much cleaner!

**Pull request merged**